bot/exts/backend/sync/_cog.py

```diff
@@ -8,7 +8,7 @@
 from bot import constants
 from bot.api import ResponseCodeError
 from bot.bot import Bot
-from bot.cogs.sync import syncers
+from . import _syncers
```

Comment:
Why is this necessary? I really dislike dot imports.

Reply:
Addressed by f150e69.

I don't feel strongly about it, but there are some nice things about relative imports:

1. They tend to be shorter
    * Easier to read (contentious, I guess)
    * Easier to write (mentioning this for completeness sake; it's not really a valid argument on its own)
    * More room to import many things without breaking into multiple lines
2. Sorting imports puts relative ones at the end. This makes it more convenient to find the most relevant objects to the current module, on the assumption that code in the same package is more relevant than objects in external packages.

They weren't importing anything deeply nested, so I didn't mind them.